moveit_ros/moveit_servo/config/panda_simulated_config.yaml

```diff
@@ -2,22 +2,13 @@
 # Modify all parameters related to servoing here
 ###############################################
 
-## Properties of incoming commands
-command_in_type: "unitless" # "unitless"> in the range [-1:1], as if from joystick. "speed_units"> cmds are in m/s and rad/s
-scale:
-  # Scale parameters are only used if command_in_type=="unitless"
-  linear:  0.4  # Max linear velocity. Unit is [m/s]. Only used for Cartesian commands.
-  rotational:  0.8 # Max angular velocity. Unit is [rad/s]. Only used for Cartesian commands.
-  # Max joint angular/linear velocity. Only used for joint commands on joint_command_in_topic.
-  joint: 0.5
-
 # Optionally override Servo's internal velocity scaling when near singularity or collision (0.0 = use internal velocity scaling)
 # override_velocity_scaling_factor = 0.0 # valid range [0.0:1.0]
 
 ## Properties of outgoing commands
 publish_period: 0.034  # 1/Nominal publish rate [seconds]
-low_latency_mode: false  # Set this to true to publish as soon as an incoming Twist command is received (publish_period is ignored)
 
+command_in_type: "speed_units"
```

Comment:
Maybe bring back a comment above this parameter since this example config is meant to help people understand their options